modules/gdnative/godot/string.cpp

```diff
@@ -538,6 +538,38 @@ godot_string GDAPI godot_string_get_slicec(const godot_string *p_self, wchar_t p
 	return result;
 };
 
+godot_array GDAPI godot_string_split(const godot_string *p_self, const godot_string *p_splitter) {
```

Comment:
oh ups, I didn't see you already had something. You don't need to go through `Variant *`, you can just do it like I wrote above. Also if you know the size beforehand you can use `resize()`, that results in better performance ;)